src/program.rs

```diff
@@ -29,8 +28,17 @@ impl Program {
                 '-' => Instruction::DecVal,
                 '.' => Instruction::Output,
                 ',' => Instruction::Input,
-                '[' => Instruction::SkipForward(bracket_map[&count]),
-                ']' => Instruction::SkipBackward(bracket_map[&count]),
+                '[' => {
+                    stack.push(count);
+                    // placeholder Instruction. iptr will be resolved later
```

Comment:
Really nitpicking but as a general note of style I tend to keep all comments in full grammatiacal English. Starting with a capital and ending with punctuation.r